src/jaxsim/api/ode.py

```diff
@@ -4,6 +4,7 @@
 import jax.numpy as jnp
 import jaxlie
 
+import jaxsim.api as js
 import jaxsim.physics.algos.soft_contacts
 import jaxsim.typing as jtp
 from jaxsim import VelRepr, integrators
```

Comment:
`jaxlie` and `integrators` are not used
```suggestion

import jaxsim.api as js
import jaxsim.physics.algos.soft_contacts
import jaxsim.typing as jtp
from jaxsim import VelRepr
```